scoring.js

```diff
@@ -0,0 +1,58 @@
+const calculateScore = (player) => {
+  switch (player.position) {
+    case 'QB':
+      return passingScore(player) + rushingScore(player)
+    case 'RB':
+      return rushingScore(player) + receivingScore(player) + returnScore(player)
+    case 'WR':
+      return rushingScore(player) + receivingScore(player) + returnScore(player)
+    case 'TE':
+      return receivingScore(player)
+    case 'K':
```

Comment:
This should be your `default` case. The requirements say "return 0 for unknown positions." `K` is just the example used in the test but there are many other possible positions and right now they wouldn't get back 0.